src/js/constants.ts

```diff
@@ -20,6 +20,9 @@ export const BANI_LENGTH_COLS = {
   extralong: 'existsBuddhaDal',
 };
 
+export const NITNEM_BANIS = [2, 4, 6, 9, 10, 20, 21, 23];
```

Comment:
Please add comments explaining which ID is which. Would help in maintenance phase.

Reply:
Pretty easy to figure out - eg. https://www.sikhitothemax.org/sundar-gutka/**2**